zscript/SpriteShadow.zc

```diff
@@ -217,6 +220,9 @@ class Z_ShadeMe : CustomInventory
 		if (countFromLastUpdate > maxMonsterShadowAgeInTicks) 
 		{
 			self.SetEnableShadow(false);
+			//OK to self-remove from inventory ?
+			Owner.TakeInventory("Z_ShadeMe", 1);
```

Comment:
Here the `Z_ShadeMe` object actually requests the `Owner` to remove itself... It works, so far.